src/OrbitQt/TrackTypeItemModelTest.cpp

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gtest/gtest.h>
+
+#include <QAbstractItemModelTester>
+#include <QModelIndex>
+#include <QSignalSpy>
+#include <QTableView>
+#include <QTest>
+#include <string>
+
+#include "QtUtils/AssertNoQtLogWarnings.h"
+#include "TimeGraphLayout.h"
+#include "TrackManager.h"
+#include "TrackManagerTest.h"
+#include "TrackTypeItemModel.h"
+
+namespace orbit_qt {
+
+class TrackTypeItemModelTest : public ::testing::Test {
+ public:
+  TrackTypeItemModelTest()
+      : Test(),
+        capture_data_(orbit_gl::TrackTestData::GenerateTestCaptureData()),
+        track_manager_(nullptr, nullptr, &layout_, nullptr, capture_data_.get()) {}
```

Comment:
Why not move these to where the fields are declared instead of having a constructor for these initializations?

Reply:
Actually I prefer that as well, I think it ended up like this due to some refactoring... thanks.